packages/web/apollo/codegen.ts

```diff
@@ -2,15 +2,24 @@ import type { CodegenConfig } from "@graphql-codegen/cli";
 
 const config: CodegenConfig = {
   overwrite: true,
-  schema: "./apollo/schema.graphql",
+  schema:
+    "https://api.thegraph.com/subgraphs/name/eonian-core/eonian-bsc-development",
```

Comment:
Maybe use preview schema?

Reply:
Decided to use more stable option to not to break the workflows